plugin.xml

```diff
@@ -47,7 +47,7 @@
         </array>
       </config-file>
 
-      <framework src="Intercom" type="podspec" spec="~> 3.2.1" />
+      <framework src="Intercom" type="podspec" spec="~> 3.2.2" />
```

Comment:
That was to fix a specific issue by requiring 3.2.1 and above IIRC. However, its still fine to bump here also 👍